api/admin/controller.py

```diff
@@ -373,7 +378,7 @@ def error_response(self, problem_detail):
 
 class WorkController(CirculationManagerController):
 
-    STAFF_WEIGHT = 1
+    STAFF_WEIGHT = 1000
```

Comment:
I changed this for ratings and it is also used for classifications, but changing it for those shouldn't matter. We originally were handling staff classifications by giving them a much higher weight than other classifications, but then we changed it so that if there are any staff classifications we use them and ignore anything else, so their weight doesn't matter except relative to other staff classifications. Maybe we'll end up doing the same thing with ratings eventually.